repl/repl_test.go

```diff
@@ -56,7 +55,7 @@ func TestREPL_printSplash(t *testing.T) {
 		t.Fatalf("failed to get user home dir: %s", err)
 	}
 
-	f, err := ioutil.TempFile(home, "")
+	f, err := os.MkdirTemp(home, "")
```

Comment:
```suggestion
	f, err := os.CreateTemp(home, "")
```

Reply:
Thanks for your review. Evans is the best command line tool I have ever used.